tests/manage/monitoring/conftest.py

```diff
@@ -808,3 +808,42 @@ def delete_target_bucket():
     ns_bucket[0].bucketclass.delete()
     ns_stores[0].delete()
     return measured_op
+
+
+@pytest.fixture
+def measure_ceph_osd_flapping(measurement_dir):
+    """
+    Delete Ceph osd, measures the time when it was
+    deleted and alerts that were triggered during this event.
+
+    Returns:
+        dict: Contains information about `start` and `stop` time for stopping
+            Ceph osd pod
+    """
+
```

Comment:
We need to have a solid teardown that will be called upon test failure. This test case is disruptive and in case it fails while OSD is down, the whole cluster is left broken for the subsequent test cases

Reply:
the test will fail if this alert won't show up 
for the teardown - I can add a check that new OSD came up after the last delete
but in a case it did not come up, what can we do?

Reply:
I think that it would be a bug.

Reply:
ok, so let's keep it as is